client/pages/home.jsx

```diff
@@ -1,10 +1,71 @@
 import React from 'react';
 import AppContext from '../lib/app-context';
+import Redirect from '../components/redirect';
 
 export default class Home extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      projects: [],
+      projectSelected: false,
+      mounted: false
+    };
+    this.handleProjectView = this.handleProjectView.bind(this);
+  }
+
+  handleProjectView() {
+    this.setState({ projectSelected: !this.state.projectSelected });
+  }
+
+  componentDidMount() {
+
+    fetch('/api/home/project', { method: 'GET' })
+      .then(res => res.json())
+      .then(result => {
+        this.setState({ projects: this.state.projects.concat(result) });
+      });
+    this.setState({ mounted: true });
+  }
+
   render() {
+    const projectList = this.state.projects.map(project =>
+      <div key={project.projectId} >
+        {
+          <div onClick={this.handleProjectView} className="card margin-top-1x">
```

Comment:
Instead of listening for click events, you should make this an anchor tag that triggers a hash change. You'll need to set the `display` to `block` as well.
```jsx
<a href={`#projects?projectId=${project.projectId}`} className="card mt-2 d-block">
```